### ISSUE TITLE
file-header should create a /*! comment, not just /*

### ISSUE THREAD

**Issue by JoshuaKGoldberg** [palantir/tslint#3738]
### Bug Report

- __TSLint version__: 5.9.1
- __TypeScript version__: 2.8.0-dev*
- __Running TSLint via__: (pick one) CLI

#### TypeScript code being linted

```ts
// code snippet
```

with `tslint.json` configuration:

```json
{
    "rules": {
        "file-header": [
            true,
            "Copyright info.",
            "Copyright info."
       ]
    }
}
```

#### Actual behavior

```typescript
/*
 * Copyright info.
 */

```

#### Expected behavior

```typescript
/*!
 * Copyright info.
 */
```

**Comment by JoshuaKGoldberg** [comment]
https://github.com/Microsoft/TypeScript/issues/22174

**Comment by rwaskiewicz** [comment]
Possibly related issue - https://github.com/palantir/tslint/issues/3696

**Comment by JoshuaKGoldberg** [comment]
@johnwiseheart nit: this should probably have the `Type: Bug` label instead of `Type: Rule Suggestion`.
